.github/workflows/rust.yml

```diff
@@ -12,24 +12,20 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v2
-
-      - name: Set variables
-        run: |
-          echo "TOOLCHAIN=$(rustup show active-toolchain | cut -d " " -f1)" >> $GITHUB_ENV
-
+        
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ env.TOOLCHAIN }}
+          toolchain: nightly
```

Comment:
Not a fan of this, as we should specify the toolchain as exact as possible, especially with nightly

Reply:
Yes I agree. That's the `nightly` version is only used for the `rustfmt` the stable channel is used for compiling and the tests.

Reply:
But even for nightly it would be nice to fix it to a specific nightly version

Reply:
Yes I agree. Changed here [b297e7b](https://github.com/gnosis/safe-client-gateway/pull/806/commits/b297e7b732582f22a0a84aa936813ad97b8cabfb)